llarp/net/route.cpp

```diff
@@ -408,6 +416,10 @@ namespace llarp::net
     Execute("/sbin/route -n delete -cloning -net 0.0.0.0 -netmask 128.0.0.0 -interface " + ifname);
     Execute(
         "/sbin/route -n delete -cloning -net 128.0.0.0 -netmask 128.0.0.0 -interface " + ifname);
+#elif __sun
+    const auto iface = GetIFAddr(ifname);
+    Execute("/usr/sbin/route delete 0.0.0.0/1 -iface " + iface->toHost());
```

Comment:
i could have also recycled the route commands but `-cloning` does nothing on SysV

Reply:
this returns a std::optional and de-referencing a std::optional is undefined behavior

Reply:
then what the hell is.....